akka-actor-typed/src/main/scala/akka/actor/typed/internal/routing/GroupRouterImpl.scala

```diff
@@ -88,22 +98,43 @@ private final class InitialGroupRouterImpl[T](
   }
 }
 
+@InternalApi
+private[routing] object GroupRouterHelper {
+  def routeesToUpdate[T](
+      allRoutees: Set[ActorRef[T]],
+      routingLogic: RoutingLogic[T],
+      preferLocalRoutees: Boolean): Set[ActorRef[T]] = {
+    val shouldFilter = preferLocalRoutees && (routingLogic match {
+        case _: RoutingLogics.RandomLogic[_]     => true
+        case _: RoutingLogics.RoundRobinLogic[_] => true
+        case _                                   => false
```

Comment:
isn't `preferLocalRoutees` already (always) `false` for consistent hashing and this pattern match isn't needed?

Reply:
you are right, but there is a place need to small fix:
in 
```
def withConsistentHashingRouting(virtualNodesFactor: Int, mapping: T => String): GroupRouterBuilder[T] = {
    import akka.actor.typed.scaladsl.adapter._
    copy(
      logicFactory = system =>
        new RoutingLogics.ConsistentHashingLogic[T](
          virtualNodesFactor,
          mapping,
          system.toClassic.asInstanceOf[ExtendedActorSystem].provider.getDefaultAddress))
  }
```
need to change to:
```
def withConsistentHashingRouting(virtualNodesFactor: Int, mapping: T => String): GroupRouterBuilder[T] = {
    import akka.actor.typed.scaladsl.adapter._
    copy(
      preferLocalRoutees = false, 
      logicFactory = system =>
        new RoutingLogics.ConsistentHashingLogic[T](
          virtualNodesFactor,
          mapping,
          system.toClassic.asInstanceOf[ExtendedActorSystem].provider.getDefaultAddress))
  }
```

Reply:
I will fix that.